.github/workflows/build.yml

```diff
@@ -23,13 +23,17 @@ jobs:
         run: |
           pip3 install scons
           scons platform=linux arch=x86_64 target=editor generate_bindings=yes
-          ldd demo/addons/godot-git-plugin/linux/*.so
+          ldd addons/godot-git-plugin/linux/*.so
+      - name: Prepare artifact
+        run: |
+          mkdir bin
+          mv addons bin/
```

Comment:
I had to add this awkward step because after removing the `demo/` nesting, I ran into an issue with the artifact, where the zip wouldn't contain the `addons/` top-level folder.

`upload-artifact@v4` doesn't give any way to configure how to zip the given path sadly, so I need to trick it by re-introducing a parent folder to zip the contents of...

Another alternative I considered is to bring back a top-level folder to contain `addons` (like `bin/addons/godot-git-plugin/`), but I find it a bit annoying.

Reply:
Well `s/bin/out/` because apparently we do have some thirdparty stuff that generates in `bin`.